src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Service/IChannelService.cs

```diff
@@ -8,4 +8,6 @@ public interface IChannelService
     Task<ChannelModel?> GetAsync(Guid id);
 
     Task<PaginatedListModel<ChannelModel>> GetListAsync(GetChannelModel options);
+
+    Task CreateAsync(ChannelUpsertModel options);
```

Comment:
UpsertAsync?

Reply:
Allow other projects to create or modify their channels